data/projectData.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "title": "About Me",
+    "preview": "https://placekitten.com/g/300/300",
+    "previewUrl": "https://github.com/aer03280/about_me",
+    "dateCreated": "2016-10-18",
+    "assigned": "class",
+    "category": "solo",
+    "description": "<p>This is a site I built with a get-to-know Ashley game. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>"
+  },
+  {
+    "title": "Trip Planner",
+    "preview": "https://placekitten.com/g/300/300",
+    "previewUrl": "https://github.com/shortaj/Group_Project-Trip-Planner",
+    "dateCreated": "2016-11-09",
+    "assigned": "class",
+    "category": "group",
+    "description": "<p>This application was created as a group assignment. It helps you select a destination for your next vacation. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>"
+
+  },
+  {
+    "title": "The Cookie Stand",
+    "preview": "https://placekitten.com/g/300/300",
+    "previewUrl": "https://github.com/aer03280/cookie-stand",
+    "dateCreated": "2016-10-24",
+    "assigned": "paid",
+    "category": "solo",
+    "description": "<p>I created this site as a class assignment for a fictitious cookie shop. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>"
+  }
+];
```

Comment:
Pass your JSON into a linter at jsonlint.com - looks like it could be the semicolon at the end of this.